tests/python/relay/test_pass_alter_op_layout.py

```diff
@@ -315,6 +315,51 @@ def expected():
     assert tvm.ir.structural_equal(a, b), "Actual = \n" + str(a)
 
 
+def test_dnnl_alter_layout_resnet():
+    """Test alternating the layout of a residual block to the dnnl optimal layout. 
+    The test case is for oneDNNV1.7. oneDNN v2.x prefers other layout.
```

Comment:
Does this test fail when oneDNN 2.x is used?
Also need to cover the shapes with `group>1`.

Reply:
This test will fail. I add another test for onednn v2.x.